### PR TITLE
Clarified incompatible node version error message.

### PR DESCRIPTION
**Pull request by hannesne**
This is in response to issue #164

### CODE REVIEW

File: src/nodejsWorker.ts [hunk 1]
```diff
@@ -15,12 +15,12 @@ function validateNodeVersion(version) {
...
-        // Unsupported version
-        } else if (major != ACTIVE_LTS_VERSION && major != CURRENT_BRANCH_VERSION) {
+        // Unsupported version note: Documentation about Node's stable versions here: https://github.com/nodejs/Release#release-plan and an explanation here: https://medium.com/swlh/understanding-how-node-releases-work-in-2018-6fd356816db4
+        } else if (major != ACTIVE_LTS_VERSION && major != SUPPORTED_ACTIVE_BRANCH) {
             message = "Incompatible Node.js version. The version you are using is "
-                    + version +
-                    ", but the runtime requires an even-numbered Active LTS or Current version (ex: 8.11.1 or 10.14.1). "
-                    + "For deployed code, change WEBSITE_NODE_DEFAULT_VERSION in App Settings. Locally, upgrade the node version used by your machine (make sure to quit and restart your code editor to pick up the changes).";
+                + version +
+                ", but the runtime requires a stable Active version, currently V8.x (LTS) and V10.x. Stable versions have an even major version number (8.x, 10.x etc) as per https://github.com/nodejs/Release#release-plan. "
```
Comment (mhoeger) [inline src/nodejsWorker.ts]: Can we change this to: "but the runtime requires an LTS-covered major version (ex: 8.11.1 or 10.14.1). LTS-covered versions have an even major version number (8.x, 10.x, etc.) as per https://github.com/nodejs/Release#release-plan. "

File: src/nodejsWorker.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 var logPrefix = "LanguageWorkerConsoleLog";
 var errorPrefix = logPrefix + "[error] ";
 var ACTIVE_LTS_VERSION = "v8";
```
Comment (mhoeger) [inline src/nodejsWorker.ts]: `ACTIVE_LTS_VERSION_0` ?

File: src/nodejsWorker.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 var logPrefix = "LanguageWorkerConsoleLog";
 var errorPrefix = logPrefix + "[error] ";
 var ACTIVE_LTS_VERSION = "v8";
-var CURRENT_BRANCH_VERSION = "v10";
+var SUPPORTED_ACTIVE_BRANCH = "v10";
```
Comment (mhoeger) [inline src/nodejsWorker.ts]: `ACTIVE_LTS_VERSION_1` ?

File: src/nodejsWorker.ts (hunk 1 above)
Comment (mhoeger) [reply src/nodejsWorker.ts]: (Note: would prefer to keep specific example versions as it is helpful for people when choosing node version in Azure)

Comment (hannesne) [reply src/nodejsWorker.ts]: Yeah that looks good, I'll make that change and commit. I agree the stable part is confusing. Had to read the medium post to understand that part about the even version numbers and stable etc... We had a long discussion in CSE about it, so I think we got most of the kinks sorted out in the message. I definitely prefer the LTS-covered nomenclature though.

File: src/nodejsWorker.ts (hunk 3 above)
Comment (hannesne) [reply src/nodejsWorker.ts]: Yep. As an aside, we should probably collapse this and current_branch_version into an array of supported versions, once v12 comes out.

Comment (hannesne) [reply src/nodejsWorker.ts]: I've gone ahead and changed that to an array of supported versions. Looks a bit cleaner.

**Comment by hannesne** [comment]
Thanks! 😀

**Pull request merged**